src/ideabox/policy/form/project_submission.py

```diff
@@ -53,11 +53,17 @@ def send_mail(self, url):
             "ideabox.policy.browser.controlpanel.IIdeaBoxSettingsSchema.project_manager_email",
             default=None,
         )
-        if email is None:
+
+        campaign_email = self.context.emails
+
+        if campaign_email is None and email is None:
             logger.warn("missing email for project submission notification")
             return
 
-        list_mail = email.split(";")
+        if campaign_email is not None:
+            list_mail = campaign_email.split(";")
+        else:
+            list_mail = email.split(";")
```

Comment:
For performance, it is better to first check if there is an email on context (campaign) before getting it from global config (registry). By doing that you can avoid a ZODB query.